src/test/java/uk/gov/companieshouse/company/profile/controller/CompanyProfileControllerTest.java

```diff
@@ -77,4 +87,14 @@ void getCompanyProfileInternalServerError() throws Exception {
                         .andExpect(content().string(""))
         ).hasCause(new RuntimeException());
     }
+
+    @Test
+    @DisplayName("Company Profile PATCH request")
+    public void callCompanyProfilePatch() throws Exception {
+        CompanyProfile request = new CompanyProfile();
+        doNothing().when(companyProfileService).update(isA(CompanyProfile.class));
```

Comment:
How about we put the request and url first, then empty line, then the when, empty line then the mockMvc call ? Would follow the given/when/then convention and make it easier to read